Yoast/wpseo-woocommerce
# Improve Schema for offers

## Summary

This PR can be summarized in the following changelog entry:

* Schema: add an `@id` attribute to every offer.
* Schema: remove the `priceValidUntil` attribute, as it's not sensible.

## Test instructions

This PR can be tested by following these steps:

* Check schema output, see every `offers` node has an `@id`.
* Check schema output, see every `offers` node has the `priceValidUntil` attribute removed.

Fixes #366 
Fixes #348
